Develop

SHCHERBANV opened the pull request.

YuriiSmolii commented on Dockerfile.mysql:
```diff
@@ -8,7 +8,7 @@ ENV MYSQL_USER=app_user
 
 # Expose port 3306 for MySQL connections
-EXPOSE 3306
+EXPOSE 3308
```
Wrong changes.

YuriiSmolii commented on docker-compose.yml:
```diff
@@ -0,0 +1,37 @@
+    build:
+      context: .
+      dockerfile: Dockerfile
+    command: sh -c "python manage.py migrate && python manage.py runserver 0.0.0.0:8080"
```
This command already included in Dockerfile.

YuriiSmolii commented on README.md:
```diff
@@ -38,10 +38,15 @@ or start on the [landing page](http://localhost:8000/)
-
+Guidelines for Docker Compose task
+1. Use commands: 
+docker pull shcherbanvasyl499/todolist-web
+docker pull shcherbanvasyl499/todolist-db
```
We don't need to pull images, docker compose will do that automatically.

YuriiSmolii commented on docker-compose.yml:
```diff
@@ -0,0 +1,37 @@
+services:
+  db:
+    hostname: db
+    build:
+      context: .
+      dockerfile: Dockerfile.mysql
```
use `image` here and specify your docker hub image

YuriiSmolii commented on docker-compose.yml:
```diff
@@ -0,0 +1,37 @@
+
+  web:
+    build:
+      context: .
+      dockerfile: Dockerfile
```
use `image` here and specify your docker hub image

YuriiSmolii reviewed (changes requested):
Remove all the cache files and keep in mind to remove all the cache in future.
Fix source configs.

SHCHERBANV commented:
I updated the docker-compose file and specified images for db/web containers, removed unnecessary commands from the 
![Знімок екрана 2024-04-10 224430](https://github.com/mate-academy/devops_todolist_docker_core_task_3_docker_compose/assets/151711805/2f25ca61-ec9e-427c-b2d7-fa8f71878dc1)
Dockerfile, but I did not change the 3308 port because the 3306 port is busy on my machine.  When the project db container and volume start automatically, looks like the web starts automatically but needs to weight for some  time or do a  hand start from the docker desktop or terminal command

NazarSheremet reviewed (approved):
Good job!
